workspaces/adoption-insights/plugins/adoption-insights/src/components/Searches/CustomTooltip.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+
+import Paper from '@mui/material/Paper';
+import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
+import { format } from 'date-fns';
+
+const CustomTooltip = ({
+  active,
+  payload,
+  label,
+  grouping,
+}: {
+  active?: boolean;
+  payload?: any[];
+  label?: string;
+  grouping?: string;
+}) => {
+  if (active && payload?.length) {
+    const date = label ? new Date(label) : new Date();
+    return (
+      <Paper
+        elevation={1}
+        sx={{
+          padding: '12px 16px',
+          boxShadow: 4,
+          borderRadius: 2,
+          border: theme => `1px solid ${theme.palette.grey[300]}`,
+          backgroundColor: 'white',
```

Comment:
This adds white color background in dark mode as well, so let us not set background color explicitly.
```suggestion

```